lib/base-compiler.ts

```diff
@@ -1011,12 +1011,13 @@ export class BaseCompiler {
         if (output.code !== 0) {
             return [{text: 'Failed to run compiler to get IR code'}];
         }
+        console.log(newOptions, output);
```

Comment:
*cough*

Reply:
Ey you found a bug! Every check passed both locally and in the CI. I'll fix it in another PR, thanks for checking the code :)